contracts/EasyAuction.sol

```diff
@@ -75,12 +75,12 @@ contract EasyAuction is Ownable {
     event UserRegistration(address indexed user, uint64 userId);
 
     struct AuctionData {
-        IERC20 sellToken;
-        IERC20 buyToken;
+        IERC20 auctioningToken;
+        IERC20 biddingToken;
         uint256 auctionEndDate;
         bytes32 initialAuctionOrder;
-        uint256 minimumParticipationBuyAmount;
-        uint256 interimSellAmountSum;
+        uint256 minimumBiddingAmount;
+        uint256 interimSumBiddedAmount;
```

Comment:
Unfortunately, the past form of bid is confusingly still bid. (This change applies also to the rest of the file.)

```suggestion
        uint256 interimSumBidAmount;
```

Reply:
Oh no... thanks!